### ISSUE TITLE
Website down

### ISSUE THREAD

**Issue by jayfield1979** [jalxob/cool-kitten#52]
Hey. You are probably already aware, but just in case; your website is down so any newcomers to your project are going to be put-off from your amazing project!

**Comment by tejasmanohar** [comment]
Yeah! I'm trying to find documentation and demo but not knowing where to start now that the site is down

**Comment by tejasmanohar** [comment]
Make a gittip if you need to. You have 1,432 stars on this repo. People will fund your domain and minimal hosting.

**Comment by robwent** [comment]
sign up here as well for the people who use flattr, I think any donators who have starred in the past you still get the cash https://flattr.com/

**Comment by rufncrus** [comment]
I found a demo for it here http://www.jqueryscript.net/demo/jQuery-Responsive-Parallax-Scrolling-Framework-Cool-Kitten/  :)
